index.html

```diff
@@ -0,0 +1,66 @@
+<html>
+  <head>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>INTENTION TIMER</h1>
+    </header>
+    <main>
+      <!-- create ALL the Left container -->
+      <section class="new-activity-section">
+        <!-- create header for left container -->
+        <h2 class="new-activity">New Activity</h2>
+        <div class="form-container">
+          <!-- create body of left container -->
+          <section class="new-act-section"></section>
+          <!-- create 3 children buttons for study, meditate, exercise inside left container -->
+        <form class="form" id="form">
+          <section class="buttons">
+            <button type="button" class="form-button study-btn" id="Study">
+              <img src="assets/study.svg" alt="light-blub" class="study">Study</button>
+            <button type="button" class="form-button meditate-btn" id="Meditate">
+              <img src="assets/meditate.svg" alt="lotus" class="meditate">Meditate</button>
+            <button type="button" class="form-button excercise-btn" id="Exercise">
+              <img src="assets/exercise.svg" alt="shoe" class="exercise">Exercise </button>
+          </section>
+          <!-- create prompt input field -->
+          <section class="prompt">
+            <label for="accomplishment">What would you like to accomplish during this time?</label>
+            <input type="text" name="accomplishment" id="accomplishment" class="input-field">
+          </section>
+          <!-- create input field for minutes and seconds -->
+            <section class="time">
+              <section class="minute">
+                <label for="minutes">Minutes</label>
+                <input type="number" name="minutes" id="minutes" class="input-field">
+              </section>
+              <section class="second">
+                <label for="seconds">Seconds</label>
+                <input type="number" name="seconds" id="seconds" class="input-field">
```

Comment:
Do we need the name populated for these "time" nodes, the DOM text is being populated by the <label>